lib/getReactFlowData.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Generates reactFlow elements data from default output syntax
+ * 
+ * @param {Object} metrics Relations between AsyncAPI docs in default output syntax
+ * @returns {Array} reactFlow elements data following 
+ */
+
+function getReactFlowData(metrics) {
+  let i=1;
+  let edgId = 1;
+  const elements = [];
+  const visitedServices = [];
+  metrics.forEach((relations,server) => {
+    const url = server.split(',');
+    const node = { id: `Server${i}`, data: { label: `${url}` }, position: { x: 250, y: 5 } };
+    elements.push(node);
+    relations.forEach((operations, channel) => {
+      operations.sub.forEach((metadata,subscriber) => {
+        const service = subscriber.replace(/\s/g,'');
+        if (!visitedServices.includes(service)) {
```

Comment:
You don't push anything to the `visitedServices`. Is code broken, do you missed about something in implementation? :)

Reply:
aah, I missed it, sorry.